less/generic.less

```diff
@@ -16,7 +16,7 @@
 // Borders and Outlines
 // ====================
 
-@focus-outline: 3px solid orange; //focus outline is displayed when accessibility is enabled
+@focus-outline: 1px; //focus outline is displayed when accessibility is enabled
```

Comment:
As per [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/outline):
> Note: The outline will be invisible if its style is not defined. This is because the style defaults to none.

Suggest either choosing an outline style for now or set to `0` if the intention is to hide it.

Going forward, we'll need to remove all `outline` CSS overrides before the browser default for `:focus` is used.